test/fixtures/output/iam/rbac/role/list-json-cloud.golden

```diff
@@ -756,6 +756,327 @@
       }
     ]
   },
+  {
```

Comment:
This is testing the same code as the YAML version of this file (it's adding no new coverage). Let's delete this test?